src/main/kotlin/org/rust/cargo/runconfig/CargoCommandConfiguration.kt

```diff
@@ -25,6 +25,7 @@ class CargoCommandConfiguration(project: Project,
 
     var command: String = "run"
     var additionalArguments: String = ""
+    var environmentVariables: Map<String, String>? = null
```

Comment:
@matklad Thank you so detailed description.

I fix it at 9169624 , and remove verbose optional `?` keyword from a few lines to apply common syntax. Would you review last PR?